### PR TITLE
Several changes to the repo:

### PR DESCRIPTION
**Pull request by zongzw**
1. Use @KAFKA label in fluentd config.
2. Add fluentd in_file and in_dummy sample.
3. Add logstash date and ruby filter sample.
4. Regulate logstash json and regex pipelines.

### CODE REVIEW

File: conf.d/fluentd/in_dummy.conf [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+#   <format>
+#     @type json
+#   </format>
+
+#   default_topic general-topic
+
+#   required_acks -1
+#   max_send_limit_bytes 10485760
+# </match>
```
Comment (potonix) [inline conf.d/fluentd/in_dummy.conf:34]: Can this be replaced with `@label @KAFKA`?

File: conf.d/fluentd/in_dummy.conf [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+#     host localhost
+#     port 20000
+#   </server>
+#   <parse>
+#     @type json
+#   </parse>
+#   <secondary>
+#     @type stdout
+#   </secondary>
+# </match>
```
Comment (potonix) [inline conf.d/fluentd/in_dummy.conf:54]: Three match represent three parallel outputs, am I right?

File: conf.d/logstash/pipelines/json/json.conf [hunk 3]
```diff
@@ -1,59 +1,13 @@
...
-        ]
-        # add_field => {
-        #     index => "${ES_INDEX}-%{+YYYY.MM.dd}"
-        # }
+    generator {
+    lines => [
+        "dummy message from generator pipeline: json."
+    ]
+    # Emit all lines 3 times.
+    count => 1
```
Comment (potonix) [inline conf.d/logstash/pipelines/json/json.conf:7]: Is this used for pressure testing?

File: conf.d/fluentd/in_dummy.conf (hunk 1 above)
Comment (zongzw) [reply conf.d/fluentd/in_dummy.conf:34]: > Can this be replaced with `@label @KAFKA`?

Right, we may uncomment them and use `@label @KAFKA` instead. Too busy to test the dummy. I originally want to use in_dummy for the smoking test.

File: conf.d/fluentd/in_dummy.conf (hunk 2 above)
Comment (zongzw) [reply conf.d/fluentd/in_dummy.conf:54]: No, only one of the `match`es will be used(that's the first one.)
I commented them all here as reference for the TODO task.

File: conf.d/logstash/pipelines/json/json.conf (hunk 3 above)
Comment (zongzw) [reply conf.d/logstash/pipelines/json/json.conf:7]: Only for smoking test to LOGSTASH.

Comment (potonix) [reply conf.d/logstash/pipelines/json/json.conf:7]: Got it.

**Pull request merged**